### DISCUSSION TITLE
“Unbound” sigue vigente en Tasks

### DISCUSSION THREAD

**Discussion by Mauriceac** [Cotalker/documentation#568] [4. Bug Reports] [answered]
A pesar de la determinación de hacer que los tasks asuman el tipo "bound", los _state machines_ con la opción "unbound" siguen creando _tasks_ con canales que no funcionan correctamente.

https://user-images.githubusercontent.com/47286214/146158961-78a3e5f0-53f9-4cfb-8975-9fae36e73ae6.mp4

**Comment by acedward** [comment]
Unbound fue deprecado. Sólo estamos soportando a máquinas de estados que estén en producción con esta configuración legacy.
De todas formas lo que estas viendo, es un efecto de los unbound no tienen canal, solo es una ilución!
¿Cómo creaste tareas unbound?

**Comment by Mauriceac** [comment]
Ese es el tema. Por lo entendido, no debería ser capaz de crear tareas unbound. Me di cuenta inicialmente porque no he actualizado el tutorial y los que lo han seguido el tutorial paso a paso han creado tareas unbound. Es decir, las opción sigue vigente igual que antes del supuesto cambio.

¿Debo suponer que "deprecado" no significa lo mismo que "no vigente"?

Maurice

**Reply by acedward** [reply]
Oh
![image](https://user-images.githubusercontent.com/3951895/147003374-e9595e27-36ba-4bbc-82ba-719419b81bf2.png)
This is a bug

**Comment by acedward** [comment, marked as answer]
https://github.com/Cotalker/documentation/issues/598
